extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -28,6 +28,12 @@ export const MULTI_CONSENT_EXPERIMENT = 'multi-consent';
 const CONSENT_STATE_MANAGER = 'consentStateManager';
 const TAG = 'consent-policy-manager';
 
+const UNBLOCK_ON_ALL_POLICY = 'consent-unblock-on-all';
+const WHITELIST_POLICY = [
```

Comment:
how about making it a map()?

Reply:
made it a map. The bad thing of using a map is that I can't do 
```
{
  UNBLOCK_ON_ALL_POLICY: true
}
```
can only do 
```
{
  '_none': true,
}
```